backend/battling/views.py

```diff
@@ -51,7 +54,17 @@ def form_valid(self, form):
         else:
             messages.success(self.request, "Battle ended! Check e-mail for results.")
 
-            # run_battle_and_send_email(battle)
+            # creator = Team.objects.filter(battle=battle, trainer=battle.creator.id)
```

Comment:
Why is this commented?

Reply:
You asked me to comment it so it wouldn't error on Heroku util we figured how to send email through there